### ISSUE TITLE
Build Server Issues: "Build Failed. Cannot connect to build server"

### ISSUE THREAD

**Issue by Teymur969** [Scirra/Construct-3-bugs#2349]
Unable to export an Xcode project due to constant failure to connect to build server.
I export my project for iOS (xcode project) > My project uploads > I get a message saying 'waiting to build' > then a constant loop of 'lost connection' / 'connection restored' for about 10 - 15 minutes until finally I get an error message saying 'Build Failed. Unable to Connect to Server'.

**Comment by shortercode** [comment]
This sounds like a project specific issue. General connections seem to be fine. Would it be possible for you to provide a project that reproduces this? A minimal one is preferable; it is likely either caused by a project setting or plugin. If you can't reproduce with a minimal then you can email your full project ( provided you are okay with that ).

**Comment by Teymur969** [comment]
Yes, you were right. The issue is project specific. I tried with my other project and it worked. Thanks. Still don't know what causes the issue though. May it be the webm file extensions?

**Comment by shortercode** [comment]
No the build server doesn't do much with static assets like music, it just places them into the project. It's likely a compilation error that is incorrectly triggering a disconnect during the build. Hence the repeated connection loss.

**Comment by Teymur969** [comment]
Now I succeed with the export but videos don't play on the emulator. The format seems right: MP4 H.264. Any idea what can cause the problem?
